modules/MusicDownloader.js

```diff
@@ -8,14 +8,20 @@ import {sterylizator} from "./Other.js";
 import {DebugSaveToFile} from "./DebugMode.js";
 import os from "os";
 import axios from "axios";
+import {exec} from "child_process";
 
-async function downloader(url) {
+async function downloader(url, votes) {
     const urlParts = url.split('?')[0].split("/");
     logger('verbose', `Wynik splita: ${urlParts}`, 'downloader');
     logger('verbose', `Wykryto: ${urlParts[3]}`, 'downloader');
     if (urlParts[3] === 'track') {
         logger('log', 'Wykryto piosenkę', 'downloader');
-        return downloadSong(url);
+        return downloadSong(url, votes);
+    } else if (votes) {
+        logger('verbose', 'Wykryto próbę pobrania piosenki nie piosenki w głosach!', 'downloader');
```

Comment:
Dziwnie to brzmi do poprawy
Proponuje: 
```suggestion
        logger('verbose', 'Wykryto próbę pobrania nie piosenki w głosach!', 'downloader');
```
(Tak wiem że sam to pisałem)